packages/react-storage/src/components/StorageBrowser/context/actions/listLocationsAction.ts

```diff
@@ -8,6 +11,8 @@ import {
   Permission,
 } from '../types';
 
+const PAGE_SIZE = 100;
```

Comment:
Confirmed offline that `1000` makes sense here

Reply:
updated